overrides/home/carousel/items.js

```diff
@@ -1,22 +1,22 @@
 export default [
     {
-      link: 'https://eyes.nasa.gov/apps/earth/#/',
-      title: 'Eyes on Earth',
-      description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae lorem et neque egestas mollis. Praesent ut porttitor mauris, a sagittis erat",
-      image: new URL('../media/seaside-background.jpg', import.meta.url).href,
-      imageAlt: 'Example image alt EIC 2024'
+      link: 'https://earth.gov/ghgcenter',
+      title: 'U.S. Greenhouse Gas Center',
+      description: "The U.S. Greenhouse Gas Center (US GHG Center) is a multi-agency effort consolidating greenhouse gas information from observations and models.",
+      image: new URL('../media/ghgc.png', import.meta.url).href,
+      imageAlt: 'Earth with Greenhouse Gas Emissions shaded in red'
     },
     {
-      link: 'https://earth.gov/mobile-climate-mapper/',
-      title: 'Card Title 2',
-      description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae lorem et neque egestas mollis. Praesent ut porttitor mauris, a sagittis erat',
-      image: new URL('../media/swamp.png', import.meta.url).href,
-      imageAlt: 'Example image alt EIC 2024'
+      link: 'https://earth.gov/sealevel',
+      title: 'Global Sea Level Rise',
+      description: 'The Global Sea Level Rise Portal is an international collaboration that offers essential information on current and project sea level rise to inform coastal communities across the globe.',
+      image: new URL('../media/global_slr.png', import.meta.url).href,
+      imageAlt: 'Coastline with crashing waves'
     },
     {
-    link: '/stories/agriculture',
-    title: 'Internal link to the Agriculture story',
-    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae lorem et neque egestas mollis. Praesent ut porttitor mauris, a sagittis erat',
-    image: new URL('../media/hyperwall-image1.png', import.meta.url).href,
-    imageAlt: 'Example image alt EIC 2024'
-  }]
\ No newline at end of file
+    link: 'https://earth.gov/sealevel/us',
```

Comment:
This link to `https://earth.gov/sealevel/us` currently redirects me to `https://earth.gov/sealevel/us/us/`, which then gives a 404. It seems to work when I enter `https://earth.gov/sealevel/us/` (trailing slash) instead.

Is the `/us` needed, at all?

```suggestion
    link: 'https://earth.gov/sealevel/us/',
```

Reply:
yeah - the /us i the federal portal and sealevel is the global. We've been asked to highlight both portals on earth.gov